Makefile

```diff
@@ -49,9 +49,14 @@ compile-multiarch:
 	$(MAKE) compile GOOS=linux GOARCH=arm
 	$(MAKE) compile GOOS=linux GOARCH=arm64
 
-test:
+test: test-unit
+test-unit:
 	@echo "=== $(INTEGRATION) === [ test ]: Running unit tests..."
-	@go test -race ./...
+	@go test -v -race $$(go list ./... | grep -v '/test/e2e')
```

Comment:
Rather than using `go list` magic, why not use build tags?

Reply:
I'm interested in this, could you elaborate a bit on how the solution would look like?

Reply:
You'd add something like this to the test file:
```go
// Copyright 2021 New Relic Corporation. All rights reserved.
// SPDX-License-Identifier: Apache-2.0

// +build integration

package operator_test
```

Then, run tests using `go test -tags integration`.